parsers/test/test_ESTADISTICO_UT.py

```diff
@@ -0,0 +1,49 @@
+from importlib import resources
+
+from requests import Session
+from requests_mock import GET, POST, Adapter
+from snapshottest import TestCase
+
+from electricitymap.contrib.lib.types import ZoneKey
+from parsers.ESTADISTICO_UT import DAILY_OPERATION_URL, fetch_production
+
+
+class TestESTADISTICO_UT(TestCase):
+    def setUp(self) -> None:
+        self.session = Session()
+        self.adapter = Adapter()
+        self.session.mount("https://", self.adapter)
+        self.session.mount("http://", self.adapter)
+        self.adapter.register_uri(
+            GET,
+            DAILY_OPERATION_URL,
+            text=resources.files("parsers.test.mocks.ESTADISTICO_UT")
+            .joinpath("production.html")
+            .read_text(),
+        )
```

Comment:
I have been using Path from pathlib instead of resources, I like that it has a shorter syntax.

Not that it really matters but it would probably be nice if we decided on which to use for consistency across the test files.

Reply:
Yeah I followed the same structure as most tests. Since this is rather a preference than an implementation design I would leave it to the dev to decide.